packages/thunderstore-api/src/delete/packageWiki.ts

```diff
@@ -6,7 +6,7 @@ import {
   PackageWikiPageDeleteRequestParams,
 } from "../schemas/requestSchemas";
 
-export async function deletePackageWikiPage(
+export function deletePackageWikiPage(
```

Comment:
What is the reasoning for removing these async/awaits? Does it change the behaviour anywhere when the apiFetch itself still awaits the request? I'm asking since if the behaviour doesn't change, there's a chance this change was done under the assumption that it would change something, which in part could cause issues elsewhere.

If this is just "removing unnecessary stuff", I guess the changes themself are fine (although a bit pointless), but they definitely shouldn't be done as part of this huge PR stack. Here they just bloat the amount of review, rebase, etc. (Again, assuming I'm not missing something.)

Reply:
Behavior is not changed as it's still the promise that is returned. Fair point that these could have been in a separate PR, not included in the stack.